### ISSUE TITLE
Problem w/ Hurricane

### ISSUE THREAD

**Issue by manitscold123** [endless-sky/endless-sky#2053]
So, I'm on the Wanderer Mid Mission ( Return to Spera Anatrusk ), which is the on where you escort the Wanderer fleet back to Spera Anatrusk, and there are 3 Derecho, 4 Tempests, but the 2 Hurricanes are missing, and that's a bit of a problem. They don't show up on the map or onscreen, and it keeps saying that the two ships are missing people to pilot. 

Ignore the Hurricane I'm piloting and the 900000 bullets in my machine gun. Don't judge. also the 8.89e+18 credits 

![screen shot 2017-01-22 at 5 03 37 pm](https://cloud.githubusercontent.com/assets/20450688/22188000/e168ca80-e0c4-11e6-8e77-ad814e2f9ab0.png)

**Comment by Amazinite** [comment]
What's that dot in the top left of the minimap?

Try selecting through all your escorts (shift+[next ship button]) and see if they can be selected, and where it points you toward where they are.

**Comment by KiLEdEnNis** [comment]
at least it ain't an Archon

**Comment by Bladewood** [comment]
@Amazinite I don't see anything wrong with a ship with zero heat.

**Comment by Nechochwen-D** [comment]
I had a flame-thrower and plasma Bactrain in my first playthrough that had zero heat.

Of course since then I have also put together a fleet of Archons and conquered the galaxy with a Fliver.

**Comment by manitscold123** [comment]
K. @Amazinite , the dot is one of my Korath Automata Warping in. The Solid fuel bar is like, 500,000 fuel, and the ship has 74000 shield health, just like a regular Hurricane. And, be happy I'm not using Archons. Thank you. At least it's not an Archon. Jesus. Christ. On. A. Bike.

**Comment by jafdy** [comment]
I can't find any mission that needs you to escort an NPC with that name. If you could find out the missions actual name that would help. Can you also attach your save file.

**Comment by KiLEdEnNis** [comment]
@jafdy I remember that mission; I thought it was when you were going to merge The Mind and those wre your escorts, but they use random Wanderer names if I remember correctly

**Comment by jafdy** [comment]
None of the missions that could hold the name "Return to Spera Anatrusk" have wanderer ships as NPCs. I asked for the save file because then I can pinpoint the mission and find out what it should be.

**Comment by Amazinite** [comment]
Pretty sure there's a mission a little farther up from "Return to Spera" that has the NPCs following you for a number of missions.

**Comment by endless-sky** [comment]
That's the "Wanderers: Mind Escorts" mission, but I can't see any reason why those Hurricanes wouldn't be showing up. Are you able to select them with the next target key? And do they follow you if you jump?

**Comment by endless-sky** [comment]
I'm not sure what could cause this. I tested the missions on my own computer, and the Hurricanes show up just fine.

**Comment by KiLEdEnNis** [comment]
so did mine a while ago on an older save file after 0.9.5 came out, so probably a mod or something

**Comment by SirStigII** [comment]
I have had a similar thing happen on the mission after that when you are escorting the scientists in a deep river. I had gotten to the destination too fast and after I landed the deep river didn't want to go into the eye and when I jumped back to see what was up it was heading away from the eye until I got close enough for it to go into formation. It still wouldn't go into the eye though, but it fixed itself when I reloaded the save, so I'm not sure what caused it.
